core/src/main/resources/hudson/security/Messages.properties

```
@@ -37,7 +37,8 @@ HudsonPrivateSecurityRealm.ManageUserLinks.Description=Create/delete/modify user
 HudsonPrivateSecurityRealm.CreateAccount.TextNotMatchWordInImage=Text didn''t match the word shown in the image
 HudsonPrivateSecurityRealm.CreateAccount.PasswordNotMatch=Password didn''t match
 HudsonPrivateSecurityRealm.CreateAccount.FIPS.PasswordLengthInvalid=Password must be at least 14 characters long
-HudsonPrivateSecurityRealm.CreateAccount.BCrypt.PasswordTooLong=Jenkins’ own user database currently only supports passwords of up to 72 bytes UTF-8 (72 basic ASCII characters, 24-36 CJK characters, or 18 emoji). Please use a shorter password.
+HudsonPrivateSecurityRealm.CreateAccount.BCrypt.PasswordTooLong.ASCII=Your password cannot be longer than 72 characters. Please use a shorter password.
+HudsonPrivateSecurityRealm.CreateAccount.BCrypt.PasswordTooLong=Your password cannot be longer than 72 characters (a-z, A-Z, 0-9, and basic punctuation; fewer when using other characters, like Chinese characters or emoji). Please use a shorter password.
```

This seems inconsistent with the other methods that don't provide the action as its implied.

e.g. `HudsonPrivateSecurityRealm.CreateAccount.FIPS.PasswordLengthInvalid`

```suggestion
HudsonPrivateSecurityRealm.CreateAccount.BCrypt.PasswordTooLong.ASCII=Your password cannot be longer than 72 characters.
HudsonPrivateSecurityRealm.CreateAccount.BCrypt.PasswordTooLong=Your password cannot be longer than 72 characters (a-z, A-Z, 0-9, and basic punctuation; fewer when using other characters, like Chinese characters or emoji).
```